### DISCUSSION TITLE
Launcher

### DISCUSSION THREAD

**Discussion by mihakralj** [mimugmail/opn-repo#45] [Ideas]
![image](https://user-images.githubusercontent.com/31756078/117058896-0a40df80-acd4-11eb-89df-eac0b20cf306.png)
![image](https://user-images.githubusercontent.com/31756078/117058969-23499080-acd4-11eb-8466-7c2a5765cc1c.png)

I don't know if this is worth packaging into a plugin, but I use this launcher to get to 'other' stuff on OPNsense that I piled up:
1. create /usr/local/widgets//widgets/launcher.widget.php
2. insert the code below and restart webui

```
<!-- Navigation buttons -->
<input class="btn btn-default" type="button" value="Unifi" onclick="location.pathname=''; location.port=8443;"/>
<input class="btn btn-default" type="button" value="ntopng" onclick="location.pathname=''; location.port=3443;"/>
<input class="btn btn-default" type="button" value="Adblocker" onclick="location.pathname=''; location.port=444;"/>
<input class="btn btn-default" type="button" value="Grafana" onclick="location.pathname=''; location.port=3000;"/>
<input class="btn btn-default" type="button" value="Kibana" onclick="location.pathname=''; location.port=5601;"/>
```

**Comment by mimugmail** [comment]
If you can customize the values .. sure :)
